internal/client.go

```diff
@@ -794,6 +798,13 @@ type (
 		onConflictOptions *OnConflictOptions
 	}
 
+	// startWorkflowResultInfo can be passed to StartWorkflowOptions to receive additional information
+	// of StartWorkflowExecution response.
+	startWorkflowResultInfo struct {
+		// Link to the workflow event.
+		Link *commonpb.Link
```

Comment:
I think it's ok, but would like to know other SDK team member's thoughts about exposing the raw proto here vs creating a high-level Go representation of a link (cc @Quinn-With-Two-Ns). If/when we expose the ability for users to set links on workflow start, what form would we accept?

Reply:
No thoughts since this isn't exposed outside of the SDK

Reply:
👍 I didn't realize it is intentionally unexposed.